test/policy.bats

```diff
@@ -52,6 +53,51 @@ SANDBOX_CONFIG="$TESTDATA/sandbox_config.json"
 	assert_log "$RESTRICTIVE_POLICY"
 }
 
+@test "allow signed image if already pulled and container created during mounting phase" {
+	start_crio
+	crictl pull "$SIGNED_IMAGE"
+	stop_crio_no_clean
+
+	SIGNATURE_POLICY="$RESTRICTIVE_POLICY" start_crio
+	POD_ID=$(crictl runp "$TESTDATA/sandbox_config.json")
+	CTR_CONFIG="$TESTDIR/config.json"
+	jq --arg CONTAINER_PATH "$CONTAINER_PATH" \
+		'.image.image = "'"$SIGNED_IMAGE"'" | .image.user_specified_image = "'"$SIGNED_IMAGE"'" | .mounts = [{
+			host_path: "",
+			container_path: $CONTAINER_PATH,
+			image: { image: "'"$SIGNED_IMAGE"'", user_specified_image: "'"$SIGNED_IMAGE"'" },
+			readonly: true
+		}]' "$TESTDATA/container_config.json" > "$CTR_CONFIG"
+
+	# Testing for container start failed not because of the signature, but of
+	# the missing command executable
+	run ! crictl create "$POD_ID" "$CTR_CONFIG" "$TESTDATA/sandbox_config.json"
+	[[ "$output" == *"unable to start container process"* || "$output" == *"No such file or directory"* ]]
+}
+
+@test "deny unsigned image if already pulled and container created during mounting phase" {
+	start_crio
+	crictl pull "$SIGNED_IMAGE"
+	crictl pull "$UNSIGNED_IMAGE"
+	stop_crio_no_clean
+
+	SIGNATURE_POLICY="$RESTRICTIVE_POLICY" start_crio
+	POD_ID=$(crictl runp "$TESTDATA/sandbox_config.json")
+	CTR_CONFIG="$TESTDIR/config.json"
+	jq --arg CONTAINER_PATH "$CONTAINER_PATH" \
+		'.image.image = "'"$SIGNED_IMAGE"'" | .image.user_specified_image = "'"$SIGNED_IMAGE"'" | .mounts = [{
```

Comment:
Same here, let's ensure that the signature verification of the mount part fails.